cmd/mistryd/mistryd_test.go

```diff
@@ -29,16 +29,20 @@ var (
 	params  = make(types.Params)
 
 	// mistry-cli args
-	host     string
-	port     string
-	username string
-	target   string
+	cliDefaultArgs CliCommonArgs
```

Comment:
I *think* you could drop `CliCommonArgs` and define the type right in here.

Reply:
Yes that would have been ok. However I need the CliCommonArgs named struct as it is the input argument to `func cliBuildJobArgs(cliArgs CliCommonArgs, ...`